src/components/RouteLayout.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Switch from 'react-router-dom/Switch';
+import Route from 'react-router-dom/Route';
+
+const RouteWithSubRoutes = route => {
+	if (!route.component) {
+		throw new Error(`route for ${route.path} must have a 'component' property`);
+	}
+	if (route.render || route.children) {
+		console.warn('route.render and route.children function not supported');
+	}
+	return (
+		<Route
+			path={route.path}
```

Comment:
Nope: https://reacttraining.com/react-router/#route.path. Any route without a path will always match. I think this might be why I'm getting an [error](https://github.com/meetup/meetup-web-platform/pull/167#issuecomment-282520535) in [web-platform-starter#173](https://github.com/meetup/web-platform-starter/pull/173).